conference/SE/pldi.yml

```diff
@@ -4,11 +4,11 @@
   rank: A
   dblp: pldi
   confs:
-    - year: 2021
-      id: pldi21
-      link: https://pldi21.sigplan.org/
+    - year: 2022
+      id: pldi22
+      link: https://pldi22.sigplan.org/
       timeline:
-        - deadline: '2020-11-20 23:59:59'
+        - deadline: '2020-11-19 23:59:59'
       timezone: AoE
-      date: June 23-25, 2020
-      place: Virtual
\ No newline at end of file
+      date: June 20-24, 2020
```

Comment:
please check the `deadline` and `date` field ~

Reply:
Thanks for pointing it out. fixed.